api/programs/programs-router.js

```diff
@@ -1,73 +1,73 @@
-const express = require('express')
-const Data = require('./programs-model')
+const express = require("express");
+const Data = require("./programs-model");
 
-const router = express.Router()
+const router = express.Router();
 
-router.get('/', (req, res) => {
+router.get("/", (req, res) => {
   Data.find()
     .then((programs) => {
-      res.status(200).json(programs)
+      res.status(200).json(programs);
     })
     .catch((error) => {
-      console.log(error)
-      res.status(500).json({ message: 'Error retrieving programs' })
-    })
-})
+      console.log(error);
+      res.status(500).json({ message: "Error retrieving programs" });
+    });
+});
 
-router.get('/:id', (req, res) => {
+router.get("/:id", (req, res) => {
   Data.findById(req.params.id)
     .then((program) => {
       program
         ? res.status(200).json(program)
-        : res.status(404).json({ message: 'Program not found' })
+        : res.status(404).json({ message: "Program not found" });
     })
     .catch((error) => {
-      console.log(error)
+      console.log(error);
```

Comment:
Remove :)

Reply:
Don't forget to handle the error and send the message back to the client once removed though!